### ISSUE TITLE
Use Hardware Acceleration on Rockchip Platforms using MPP

### ISSUE THREAD

**Issue by srimanachanta** [PhotonVision/photonvision#1497]
Currently, frame capture is done using the OpenCV decode function (both in cscore and libcamera backends) which is software only. At the minimum, [Rockchip MPP](https://github.com/rockchip-linux/mpp) and OMX based backends should be implemented to use hardware acceleration for frame decode.

**Comment by mcm001** [comment]
As long as it doesn't decide to randomly go kaboom like the RGA resize last year...

**Comment by Gold856** [comment]
Yeah, so [the mpp repo got DMCA'd](https://hackaday.com/2026/01/05/github-disables-rockchips-linux-mpp-repository-after-dmca-request/) and I really don't want to mess with that...
